src/nef_operations/operations.py

```diff
@@ -227,5 +227,60 @@ def get_ue_handover_event(ip, port, ue_supi, token):
     )
     
     print("Get UEs Handovers Information:", response.text)
+    if response.status_code not in [200, 201, 409]:
+        response.raise_for_status()
+
+def subscribe_qos_event (ip, port, callback_url, token):
+
+    url = f"http://{ip}:{port}/nef/api/v1/3gpp-as-session-with-qos/" \
+        "v1/netapp/subscriptions"
+        
+    headers = {}
+    headers["accept"] = "application/json"
+    headers["Authorization"] = "Bearer " + token
+    headers["Content-Type"] = "application/json"
+    
+    monitoring_payload = {
+        "ipv4Addr": "10.0.0.0",
+        "ipv6Addr": "0:0:0:0:0:0:0:0",
+        "macAddr": "22-00-00-00-00-00",
+        "notificationDestination": callback_url,
+        "snssai": {
+            "sst": 1,
+            "sd": "000001"
+        },
+        "dnn": "province1.mnc01.mcc202.gprs",
+        "qosReference": 9,
+        "altQoSReferences": [
+            0
+        ],
+        "usageThreshold": {
+            "duration": 0,
+            "totalVolume": 0,
+            "downlinkVolume": 0,
+            "uplinkVolume": 0
+        },
+        "qosMonInfo": {
+            "reqQosMonParams": [
+            "DOWNLINK"
+            ],
+            "repFreqs": [
+            "EVENT_TRIGGERED"
+            ],
+            "latThreshDl": 0,
+            "latThreshUl": 0,
+            "latThreshRp": 0,
+            "waitTime": 0,
+            "repPeriod": 0
+        }
+    }
```

Comment:
This monitoring payload comprises important monitoring information such as:
- ipv4Addr
- ipv6Addr
- macAddr
- dnn

Even though the callback_url is configured by the client (through the mini_api_configuration robot script), the parameters mentioned above are static. Therefore, we must adapt this test to allow our testing clients to configure those parameters. I see 2 ways of doing this:
- pass these parameters during the mini_api_configuration robot script
- pass the entire payload in the POST request to the MiniAPI (perhaps this approach is better)

I wonder if we should omit the mini_api_configuration script entirely and pass all parameters in the POST payloads used to trigger the tests execution ...